jenkins-jobs/pipelines/release-pipeline.groovy

```diff
@@ -333,6 +333,17 @@ node('Slave') {
             }
         }
 
+        stage('Check missing backports') {
+            if (!DRY_RUN) {
+                dir(DEBEZIUM_DIR) {
+                    def rc = sh(script: "github-support/list-missing-commits-by-issue-key.sh", returnStatus: true)
+                    if (rc != 0) {
+                        error "Error, there are some missing backport commits."
```

Comment:
@Naros Instead of hard stop I'd recommend to use https://www.jenkins.io/doc/pipeline/steps/pipeline-input-step/#input-wait-for-interactive-input as I can imagine there will be valid exceptions to the results provided by the script.
But it is better to tune with release.